contributors/devel/development.md

```diff
@@ -279,20 +279,42 @@ To build binaries for all platforms:
 make cross
 ```
 
+#### Install etcd
+```
+hack/install-etcd.sh
+```
+
+#### Verification
+```
+# Run all hack/verify-*.sh scripts
+make verify
+```
+
 #### Test
 
 ```sh
 cd $working_dir/kubernetes
 
+# Run all hack/update-*.sh scripts
+# For small change, run "make verify" and then relevant scripts, for example:
+# hack/update-gofmt.sh (when adding new files)
+# hack/update-bazel.sh (when updating dependencies / imports)
+make update
+
 # Run every unit test
 make test
 
 # Run package tests verbosely
 make test WHAT=pkg/util/cache GOFLAGS=-v
+# Or use bazel (quote "..." when using zsh)
+bazel test //pkg/util/cache/...
 
 # Run integration tests, requires etcd
 make test-integration
 
+# (Optional) Build e2e test binaries
+make all WHAT=test/e2e/e2e.test
```

Comment:
I will write a paragraph summarizing this. I will have another section specifically for local cluster's e2e testing. Thanks for the clarification!